<launch> 
    <param name="/use_sim_time" value="true" />   
    
    <!-- Launch turtle bot world -->
    <include file="$(find turtlebot_gazebo)/launch/turtlebot_world.launch"/>       
 
    <!-- Launch stopper node -->
    <node name="wander_bot_node" pkg="wander_bot" type="wander_bot_node.py" output="screen">
    	<param name="forward_speed" type="double" value="0.6"/>
    	<param name="rotation_speed" type="double" value="0.9"/>
    	<param name="min_scan_angle" type="double" value="-40"/>
    	<param name="max_scan_angle" type="double" value="40"/>
    	<param name="min_dist_from_obstacle" type="double" value="0.8"/>
    </node>
</launch>
